src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,13 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    public List<ReadOnlyPerson> sortedView() {
+        Collections.sort(internalList, new PersonComparator());
+        List<ReadOnlyPerson> sortedList = new ArrayList<ReadOnlyPerson>();
+        sortedList.addAll(internalList);
+        return sortedList;
+    }
+
```

Comment:
I see an additional blank line here. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.